.circleci/config.yml

```diff
@@ -21,6 +21,9 @@ executors:
   machine:
     machine:
       image: ubuntu-1604:201903-01
+  cuda:
+    docker:
+      - image: ymotongpoo/cuda:10.2
```

Comment:
I assume this image is built from the Dockerfile below?

It'd be nice to publish this automatically but we probably wouldn't to publish it on every commit. CircleCI does support conditional steps but I'm not sure what options are available - e.g. it'd be nice to just publish this if a file in the `receiver/cudareceiver/build` path changed or something along those lines?

It would also be nice to publish this to the https://hub.docker.com/u/otel account if possible.

Do you have more thoughts on this @owais?

Reply:
This is exactly what I wanted to know. Yes the image (`ymotongpoo/cuda:10.2`) is built from the Dockerfile in 'receiver/cudareceiver`, but I wasn't sure how I can push the image to `otel` repo.

@owais could you give some advise on Docker image registration via this PR?